Add chat, forum and other changes to Media (#11)
dobromir-hristov/vuecommunity: docs/guide/community/social-media.md

# Media

## Official Chat

> Official Vue.js chat on Discord gathers thousands of developers from around the world. The most popular channels are #need-help where you can share your issues with code and #general if you have some questions about Vue.js itself. Other channels are dedicated to particular topics, such as #best-practices, #vuex, #jobs or #mobile-apps. 

Reviewer: ```suggestion
Official Vue.js chat on Discord gathers thousands of developers from around the world. Among the many channels, the most popular ones are #need-help where you can share your develompent related issues and #general if you have some questions about Vue.js itself. Other channels are dedicated to particular topics, such as #best-practices, #vuex, #jobs or #mobile-apps. 
```

> The main language on the chat is English, but there are also rooms for conversations in other languages. On the other hand, many bigger libraries for Vue.js have their own Discord servers or chats on other platforms, so they don't have their own channel on VueLand. We list them in one of the next sections.

Reviewer: ```suggestion
The chats main language is English, but there are also rooms for conversations in other languages. Many bigger Vue.js libraries have their own Discord servers or chats on other platforms, so they don't have their own channel on VueLand. To learn about them, take a look at the [Other Platforms](./social-media.md#other-platforms) section.
```

> Remember that people you discuss with are not obligated to provide help and that they're not there all the time, so it can take a while or ten to receive answer. It may also vanish under other messages. If your problem is more complex and can take longer to solve, it may be wiser to share it on the forum and then link it on the chat.

Reviewer: ```suggestion
Remember that people you discuss with are not obligated to provide help and that they're not there all the time, so it can take a while to receive an answer. It may also vanish under other messages. If your problem is more complex and can take longer to solve, it may be wiser to share it on the forum and then link it on the chat.

> Occasionally the chat's team organizes Q&A sessions with important members of Vue.js community. Each such Q&A has its specific topic and two or more special guests answering the questions.

Reviewer: ```suggestion
Occasionally the chat's team organizes Q&A sessions with important members of the Vue.js community. Each such Q&A has its specific topic where the guests answer questions, asked by other members. 
```

Author: You don't signify here that there is always more than one guest, which is one of our main differences with Reactiflux Q&As. The `guests` itself  is not enough cause it can just mean multiple events with one guest per each.

Author: We now use "members" both in terms of people answering and asking questions. Replace one with a synonym and we're fine.

#### Useful links

* [Vue.js chat on Discord](https://chat.vuejs.org)
* [Vue.js chat's website](https://vue-land.js.org)

## Official Forum

Official Vue forum is a SEO friendly alternative to the chat. It's especially helpful if you expect to take longer to get the answer and can afford to wait for it. The forum's team is managed by Thorsten Lünborg, but you can also find help from other Core Team members there from time to time.

#### Useful links

* [Vue.js forum on Discourse](https://forum.vuejs.org)

## Stack Overflow

[Stackoverflow's Vue.js](https://stackoverflow.com/tags/vue.js/info) section is a common place for people to ask questions related to programming with Vue.

## Newsletters

> Vue.js News is an official Vue.js weekly newsletter, curated by Damian Dulisz, member of the Vue.js Core Team. There's also a video version of it, prepared by Adam Jahr and Gregg Pollack from VueMastery learning platform. 

Reviewer: ```suggestion
Vue.js News is an official Vue.js weekly newsletter, curated by Damian Dulisz, member of the Vue.js Core Team. There's also an audio podcast version of it, prepared by Adam Jahr and Gregg Pollack from VueMastery learning platform.
```
